source/test-suite/test-rope.lisp

```diff
@@ -0,0 +1,42 @@
+(in-package :caten/test-suite)
```

Comment:
If you create `test-rope.lisp`, you have to add this file into `caten/test-suite/caten.test-suite.asd`. otherwise lisp wont recognise this